test/unit/test_radioactive_decay.py

```diff
@@ -0,0 +1,31 @@
+from festim import RadioactiveDecay
+import pytest
+
+
+def test_init():
```

Comment:
Dont know if its worth adding doc strings on these tests given how simple they are

Reply:
That's what I thought